playbooks/tasks/portal-accounts-test-user-prepare.yml

```diff
@@ -0,0 +1,103 @@
+---
+# Create and Save Accounts Skynet API Key to .env file and to secure storage
+# (LastPass).
+
+- name: Set Accounts variables
+  set_fact:
+    accounts_test_user_email: "{{ webportal_server_config.accounts_test_user_email }}"
+    accounts_test_user_password: "{{ webportal_server_config.accounts_test_user_password }}"
+    accounts_api_url: "https://account.{{ ansible_host }}/api"
+
+- name: Ensure portal Accounts user exists
+  ansible.builtin.uri:
+    url: "{{ accounts_api_url }}/user"
+    method: POST
+    body:
+      email: "{{ accounts_test_user_email }}"
+      password: "{{ accounts_test_user_password }}"
+    body_format: json
+    status_code:
+      - 200
+      - 400
+  vars:
+    expected_error_message: 'identity already belongs to an existing user'
+  register: create_user_result
+  # If the user already exists we expect status code 400 and an expected
+  # error message in the response. If the user didn't exist before, user is
+  # created, we expect status code 200 and skynet-token in the response.
+  # Otherwise we fail.
+  failed_when: >-
+    (
+      create_user_result.status == 400
+      and
+      create_user_result.json.message != expected_error_message
+    )
+    or
+    (
+      create_user_result.status == 200
+      and
+      create_user_result.skynet_token is not defined
+      and
+      create_user_result.skynet_token | length < 1000
```

Comment:
The JWT token length is 1034 B (as I have seen),
if we get it shorter then 1000 B most probably we have something misconfigured and we will right here to avoid debugging overhead when some later task fails because of this.

Reply:
can you add a comment so we remember that?

Reply:
Just keep in mind that the JWT token length can change from user to user. It includes the user's email, so if I get my hands on an email like `a@b.c`. We might also adjust its content in the future.

But all of this is just for context - the current code is fine.